### PR TITLE
add command line flags 

### PR DESCRIPTION
**Pull request by arsssen**
add command line flags  to modify the comment text and to allow comments without leading spaces

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -13,16 +12,24 @@ import (
 	"os"
 	"path/filepath"
 	"strings"
+
+	"github.com/dave/dst"
+	"github.com/dave/dst/decorator"
 )
 
-const godocCommentFormat = "// %s missing godoc"
+var godocCommentFormat = "// %s missing godoc"
```
Comment (DimitarPetrov) [inline main.go]: I just changed the default comment to include punctuation. Please pull upstream.

File: main.go [hunk 2]
```diff
@@ -13,16 +12,24 @@ import (
...
 	wd, err := os.Getwd()
 	if err != nil {
 		log.Fatalf("error getting current working directory: %v", err)
 	}
-
+	commentFormat := flag.String("format", "// %s missing godoc", "comment format")
+	allowNoSpace := flag.Bool("allow_no_space", false, "allow comments without space between // and type name")
+	flag.Parse()
+	godocCommentFormat = *commentFormat
+	godocAllowNoSpace = *allowNoSpace
```
Comment (DimitarPetrov) [inline main.go]: Can you extract those in a init function as it is the idiomatic way to handle flags?

Comment (DimitarPetrov) [reply main.go]: Also, in the `commentFormat` flag, use the corresponding constant as a default value.

Comment (DimitarPetrov) [reply main.go]: `--allow_no_space` is not so intuitive as it uses underscores. Please use `--allow-no-space` dash notation.

File: main.go [hunk 3]
```diff
@@ -13,16 +12,24 @@ import (
...
 	"path/filepath"
 	"strings"
+
+	"github.com/dave/dst"
+	"github.com/dave/dst/decorator"
 )
 
-const godocCommentFormat = "// %s missing godoc"
+var godocCommentFormat = "// %s missing godoc"
+var godocAllowNoSpace = false
```
Comment (DimitarPetrov) [inline main.go]: Do we really need to make a flag for this? Can we just allow comments without space between the // and the type name by default as it is a valid godoc.

**Review by DimitarPetrov — CHANGES_REQUESTED** [review]
Thanks for your contribution! Here are some improvements/simplifications prior to merging this.

File: main.go (hunk 3 above)
Comment (arsssen) [reply main.go]: not really, added it not to change the default behavior, but will remove

File: main.go (hunk 2 above)
Comment (arsssen) [reply main.go]: done

**Review by DimitarPetrov — APPROVED** [review]

**Pull request merged**